### DISCUSSION TITLE
Running OpenDDS in Java Framework (Spring Boot)

### DISCUSSION THREAD

**Discussion by hendraprawira** [OpenDDS/OpenDDS#4438] [Q&A] [answered]
i've already implement OpenDDS to Spingboot with Gradle, but there some problem when i load arguments -DCPSConfigFile rtps_disc.ini while i running gradle bootrun, i've alrerady implement many ways, but there still didnt load the arguments line, theres any one can help me? thankyou very much

**Comment by mitza-oci** [comment, marked as answer]
Did you see any log messages from OpenDDS relevant to this?
It would help if you could share some code -- is your application's usage of `TheParticipantFactory` equivalent to what's in tests like `java/tests/messenger/publisher/TestPublisher.java`?

**Reply by hendraprawira** [reply]
Ok thankyou sir, its done for the issues.
